ui/repl.c

```diff
@@ -595,8 +595,21 @@ int main(int argc, char *argv[])
 {
     uv_setup_args(argc, argv); // no-op on Windows
 #else
+
+#if defined(_P64) && defined(JL_DEBUG_BUILD)
```

Comment:
other than the current only place you're calling it, is anything about this specific to 64 bit or debug build? fine to leave it this way for now, just curious. we might need this elsewherr later, at which point we can change or remove the ifdef.